Mage/src/main/java/mage/filter/common/FilterSuspendedCard.java

```diff
@@ -0,0 +1,50 @@
+package mage.filter.common;
+
+import mage.MageObject;
+import mage.abilities.keyword.SuspendAbility;
+import mage.counters.CounterType;
+import mage.filter.FilterCard;
+import mage.filter.predicate.mageobject.AbilityPredicate;
+
+/**
+ * @author skiwkr
+ */
+public class FilterSuspendedCard extends FilterCard {
+
+    protected FilterCard cardFilter;
+
+    public FilterSuspendedCard() {
+        this("suspended card");
+    }
+
+    public FilterSuspendedCard(String name) {
+        super(name);
+        cardFilter = new FilterCard();
+        cardFilter.add(new AbilityPredicate(SuspendAbility.class));
+        cardFilter.add(CounterType.TIME.getPredicate());
```

Comment:
this is wrong. Need to add predicates directly to the filter, there shouldn't be a separate cardFilter object at all.